packages/suite/src/support/messages.ts

```diff
@@ -8625,7 +8625,7 @@ const messages = defineMessagesWithTypeCheck({
     TR_STAKE_STAKING_IS: {
         id: 'TR_STAKE_STAKING_IS',
         defaultMessage:
-            "Staking involves temporarily locking your Ethereum assets to support the blockchain's operation. In return, you'll earn additional Ethereum as a reward.",
+            "Staking involves temporarily locking your {network} assets to support the blockchain's operation. In return, you'll earn additional {network} as a reward.",
```

Comment:
```suggestion
            "Staking involves temporarily locking your {networkSymbol} to support the blockchain's operation. In return, you'll earn additional {networkSymbol} as a reward.",
```
ETH, SOL + remove `assets`

Reply:
done, https://github.com/trezor/trezor-suite/commit/21e5cd1751b42e558b43fe4194bcdd715682675c